astronomer/airflow/version_check/templates/update-available.html

```diff
@@ -16,10 +16,18 @@
       was released on
       <time datetime="{{ cea_update_available.date_released.isoformat() }}">{{ cea_update_available.date_released.strftime("%Y-%m-%d") }}</time>
       {%- if cea_update_available.level == "critical" %}
-        and this is a critical fix
-      {%- endif -%}
-      .
+        and this is a critical fix.
+      {%- endif %}
     </p>
+    {% if cea_update_available.days_to_eol is not none %}
```

Comment:
I'm not sure we should just lump these in with the update available warning. What if you are on the latest version and it's about to be EOL?

@cmarteepants do you think either of these should be dismissable? I think the 30 days yes, EOL no?

Reply:
Was thinking about this more. What if we let you dismiss it for let's say 1 week for 6 months after it's EOL, and then after 6 months no longer allow you to dismiss it?

Reply:
Works for me. Should as add a way to opt-out universally from these EOL warnings, without disabling the whole checker?

(@sunank200, sorry this wasn't all defined before you got started)

Reply:
Yeah that makes sense

Reply:
I have added the suggested change. Please check.